app/src/main/java/com/keylesspalace/tusky/fragment/preference/PreferencesFragment.kt

```
@@ -33,17 +34,17 @@ class PreferencesFragment : PreferenceFragmentCompat() {
 
         addPreferencesFromResource(R.xml.preferences)
 
-        val themePreference: Preference = findPreference("appTheme")
-        themePreference.icon = IconicsDrawable(themePreference.context, GoogleMaterial.Icon.gmd_palette).sizePx(iconSize).color(ThemeUtils.getColor(themePreference.context, R.attr.toolbar_icon_tint))
+        val themePreference: Preference? = findPreference("appTheme")
+        themePreference?.icon = IconicsDrawable(themePreference?.context, GoogleMaterial.Icon.gmd_palette).sizePx(iconSize).color(ThemeUtils.getColor(themePreference?.context as Context, R.attr.toolbar_icon_tint))
```

Again, if it's not there we silently ignore the error, not good